core/src/subgraph/instance_manager.rs

```diff
@@ -747,6 +772,16 @@ where
             })?;
         section.end();
 
+        let section = ctx
+            .host_metrics
+            .stopwatch
+            .start_section("entity_cache_evict");
+        cache.evict(*ENTITY_CACHE_SIZE);
+        section.end();
+
+        assert!(ctx.state.entity_lfu_cache.is_empty());
```

Comment:
This would say `assertion failed: ctx.state.entity_lfu_cache.is_empty()` which lets us know that this is the assert being hit, which is really all we need for debugging. For clarity, I'll add a comment here.